setup.py

```diff
@@ -24,6 +25,9 @@ def read(name):
     with open('src/defs.pyx', 'w') as f:
         f.write(defs)
 
+if platform.system() == 'Darwin':
+    os.environ['CFLAGS'] = '-std=c++11'
```

Comment:
I haven't tested this option yet, but it seems that its better to use ``extra_compile_args``?

http://stackoverflow.com/questions/6928110/how-may-i-override-the-compiler-gcc-flags-that-setup-py-uses-by-default
http://stackoverflow.com/questions/1676384/how-to-pass-flag-to-gcc-in-python-setup-py-script